src/main/java/seedu/address/model/person/Pronoun.java

```diff
@@ -39,7 +39,7 @@ public static boolean isValidPronoun(String test) {
 
     @Override
     public String toString() {
-        return pronoun;
+        return pronoun + "/";
```

Comment:
Have we taken care of removing the last "/" when this `toString()` method is used? Just want to make sure this change doesn't affect our UI.

Reply:
Nope, thanks for pointing this out. 
I tried changing the pronoun toString() to just return itself and added this to the Person toString():
`pronouns.stream().limit(1).forEach(builder::append);`
`pronouns.stream().skip(1).forEach(pronoun -> builder.append("/" + pronoun));`
This will print out the pronouns in alphabetical order, so not by the order added.
Would this be better?

Reply:
there is a also check to see if there is only one pronoun

Reply:
Yes, this will work better.